packages/devextreme/js/__internal/utils/version.ts

```diff
@@ -35,16 +30,11 @@ export function parseVersion(version: string): Version {
 }
 
 export function assertDevExtremeVersion(packageName: string, version: string): void {
-  assertedVersions.push({
-    packageName,
-    version,
-  });
+  config({ assertedVersions: [...config()?.assertedVersions ?? [], { packageName, version }] });
```

Comment:
What if we add `assertedVersions: []` to the default config object:
https://github.com/DevExpress/DevExtreme/blob/25_1/packages/devextreme/js/__internal/core/m_config.ts#L35

Then remove the `?` from the `GlobalConfig` type prop and avoid these `?? []` checks?

Let's try if this works. It may not, because for some reason all the properties in the `GlobalConfig` type are optional. But worth a try, isn't it?